recipe/recipe.yaml

```diff
@@ -1,22 +1,22 @@
 context:
   name: "siscone"
-  version: "3.1.1"
+  version: "3.1.2"
 
 package:
   name: ${{ name }}
   version: ${{ version }}
 
 source:
   url: https://gitlab.com/fastjet/siscone/-/archive/siscone-${{ version }}/siscone-siscone-${{ version }}.tar.gz
-  sha256: 270cc6821f1614a69a60a21224b6415081f02374256a2f1c70d39ec717fd96a0
-  patches: remove-getopt-required.patch
+  sha256: 689696ca32682f39c0999a66117e673f71b41f41debb46111904f6acdcdb2e6b
+  patches: 0001-use-imported-getopt.patch
 
 build:
-  number: 4
+  number: 0
   script:
     # Get an updated config.sub and config.guess
     - if: unix
-      then: cmake "${CMAKE_ARGS}" -DSISCONE_ENABLE_DEBUG=OFF -DCMAKE_INSTALL_PREFIX="${PREFIX}" -S . -B build
+      then: cmake "${CMAKE_ARGS}" -DSISCONE_BUILD_EXAMPLES=OFF -DSISCONE_ENABLE_DEBUG=OFF -DCMAKE_INSTALL_PREFIX="${PREFIX}" -S . -B build
```

Comment:
@lgray couldn't this have been left on but then just filtered on

```
    - if: unix and target_platform == build_platform
      then: cmake ...
    - if: unix and target_platform != build_platform
      then: cmake ... -DSISCONE_BUILD_EXAMPLES=OFF ...
```

? Or are there other issues with that?

Reply:
There's no need to build the examples on any unix platform since the tests build them separately regardless of cross compiling.
The windows build needs to build them since it ships the executables to test with the packaged dlls.